controllers/apply/form-router-next/lib/form-model.js

```diff
@@ -91,11 +105,25 @@ class FormModel {
             section.progress = {
                 slug: section.slug,
                 label: section.shortTitle || section.title,
-                status: sectionStatus(this.validation)
+                status: sectionStatus(validation)
             };
 
+            function hasFeaturedErrors() {
```

Comment:
Flags if a section contains any fields with featured errors so that we can open the detail summary by default.